contrib/init/sysvinit-debian/docker

```diff
@@ -107,7 +107,7 @@ case "$1" in
 			--pidfile "$DOCKER_SSD_PIDFILE" \
```

Comment:
the `DOCKER` var in the line above this should be `DOCKERD` now :smile: (both here, at the top of this file, and in `sysvinit-debian/docker.default`)